basic_nodes/src/main/java/club/mondaylunch/gatos/basicnodes/ListHeadTailNodeType.java

```diff
@@ -0,0 +1,66 @@
+package club.mondaylunch.gatos.basicnodes;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+
+import club.mondaylunch.gatos.core.data.DataBox;
+import club.mondaylunch.gatos.core.data.DataType;
+import club.mondaylunch.gatos.core.data.ListDataType;
+import club.mondaylunch.gatos.core.data.OptionalDataType;
+import club.mondaylunch.gatos.core.graph.connector.NodeConnector;
+import club.mondaylunch.gatos.core.graph.type.NodeType;
+
+public class ListHeadTailNodeType extends NodeType.Process {
+    @Override
+    public Map<String, DataBox<?>> settings() {
+        return Map.of(
+            "head_mode", DataType.BOOLEAN.create(true)
```

Comment:
I had to read the code to work out that this makes the node take the last element instead of the first element.
Users can't/won't read the code, so this name has to be more descriptive.

Reply:
Now uses an ENUM